extensions/positron-code-cells/src/parser.ts

```diff
@@ -64,8 +64,10 @@ function getJupyterMarkdownCellText(cell: Cell, document: vscode.TextDocument):
 	return `%%markdown\n${text}\n\n`;
 }
 
-const pythonIsCellStartRegExp = new RegExp(/^\s*#\s*%%/);
-const pythonMarkdownRegExp = new RegExp(/^\s*#\s*%%[^[]*\[markdown\]/);
+// Spaces can not occur before #
```

Comment:
Do you know if there's a spec or some motivation for these? I see that the VSCode Jupyter extension also disallows spaces before `#` ([src](https://github.com/microsoft/vscode-jupyter/blob/09f9fe3d1a847992115200ba85c15f8fef14d76a/src/webviews/webview-side/react-common/settingsReactSide.ts#L23-L24)) but would be nice to have something more concrete.

Reply:
There's not a spec or motivation for this (that I know of), but all the versions I describe below do not work if there is a space before the `# `